views/deductions/climate-action-incentive.pug

```diff
@@ -16,22 +16,22 @@ block content
     div
       details.
         <summary><span>#{__('Check if you live in a small and rural community')}</span></summary>
-        <p>#{__('You live in a small and rural community if you live outside of:')}</p>
+        <p>#{__('You live in a small and rural community if you ')}<strong>#{__('do not')}</strong>#{(' live in:')}</p>
```

Comment:
It looks like the 'live in:' didn't get tagged into the locale files.

Reply:
Right you are

Reply:
Fixed.